dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -375,6 +373,32 @@ def test_unalikeability(self):
         profile.update(df_categorical)
         self.assertEqual(profile.unalikeability, 2*(10 + 15 + 6)/90)
 
+    def test_top_k_categories_change(self):
+        options = CategoricalOptions()
```

Comment:
See comment below on line 388, can just keep this `options` and manually set the `top_k_categories` for each case